c2corg_ui/static/js/user.js

```diff
@@ -120,4 +120,13 @@ app.UserController.prototype.hasEditRights = function(doctype, options) {
   return this.auth.hasEditRights(doctype, options);
 };
 
+
+/**
+ * @return {boolean}
+ * @export
+ */
+app.UserController.prototype.isModerator = function() {
+  return this.auth.isModerator();
+};
```

Comment:
I like to keep the logic in templates to a minimum because the code in templates in not compiled and is harder to test.